examples/fabric-admin/rpc/RpcServer.cpp

```diff
@@ -167,8 +187,8 @@ class FabricAdmin final : public rpc::FabricAdmin, public IcdManager::Delegate
         chip::Platform::Delete(data);
     }
 
-    // Modifications to mPendingKeepActiveTimesMs should be done on the MatterEventLoop thread
-    std::map<chip::NodeId, uint32_t> mPendingKeepActiveTimesMs;
+    // Modifications to mPendingCheckIn should be done on the MatterEventLoop thread
```

Comment:
why?

Reply:
otherwise we would need a mutex protecting this data. Added a comment

Reply:
ah, makes sense. OK, thanks.

Reply:
Actually, is there an assertion for this? It's like assert check stack locked or something?

Reply:
I'll add that in the check, but I think the comment helps communicate why